README.md

```diff
@@ -1,4 +1,4 @@
-# wallet-address-validator
+# road2crypto-wallet-address-validator
 
```

Comment:
Can you try to extract the validation mechanism from the `web3-validator` package to see if we can get rid of it?

Reply:
Make sure to keep the same order of wallet validation I had

Reply:
Let me see

Reply:
I tried it, found the function `isAddress` which we need in the package [file](https://github.com/web3/web3.js/blob/4.x/packages/web3-validator/src/validation/address.ts), but as soon as I added the file and the files it needs then, then these files also needs other files and its keep going, feeling like almost copying the whole code here, I guess we can keep using the package.

Reply:
that's fine I'll refactor it later